maintainers/README.md

```diff
@@ -0,0 +1,79 @@
+# Nix maintainers team
+
+## Motivation
+
+The goal of the team to help other people to contribute to Nix.
+
+## Members
+
+- Eelco Dolstra (@edolstra) – Team lead
+- Théophane Hufschmitt (@thufschmitt)
+- Valentin Gagarin (@fricklerhandwerk)
+- Thomas Bereknyei (@tomberek)
+- Robert Hensing (@roberth)
+
+## Meeting protocol
+
+The team meets twice a week:
```

Comment:
Are the meetings open for anybody to join and listen in and contribute?

Reply:
We didn't really settle on that afaik (so thanks for asking, that way we have to).
I personally would say they are since there's no reason for them not to be.

Reply:
We've only barely established a workflow that's _starting_ to be productive, so I'm not looking forward to us having to explain such things.

> contribute

A main goal of the team is already to process contributions in the form of a PRs. Prioritizing contributions in the form of questions during the meeting would not be fair and not be as transparent as GitHub comments, which don't fade into obscurity as much as meeting notes do, and are more precise and complete than meeting notes.

These aren't reasons not to allow anyone to listen in, so I guess what we need to do is manage expectations.